Makefile

```diff
@@ -95,6 +95,12 @@ E2E_TEST_FLAGS = "--skip-deletion=false" -timeout 25m # See README.md, default g
 # see target "image-build"
 IMAGE_BUILD_FLAGS ?= --build-arg USE_LOCAL=false
 
+# Prometheus-Unit Tests Parameters
+PROMETHEUS_CONFIG_YAML = ./config/monitoring/prometheus/apps/prometheus-configs.yaml
+PROMETHEUS_CONFIG_DIR = ./config/monitoring/prometheus/apps/
+GENERATED_ALERT_DIR = ./tests/prometheus_unit_tests/
+CRITICAL_SEVERITY="critical"
```

Comment:
@zdtsw from previous PR [comment](https://github.com/red-hat-data-services/rhods-operator/pull/366#discussion_r1827968946) removed the params because we are no longer using containers